controllers/vspherecluster_reconciler.go

```diff
@@ -351,6 +353,23 @@ func (r clusterReconciler) reconcileDeploymentZones(ctx *context.ClusterContext)
 	return true, nil
 }
 
+func shouldIncludeZone(zone infrav1.VSphereDeploymentZone, cluster *infrav1.VSphereCluster) bool {
+	// expects cluster to be annotated
+	// vsphere.infrastructure.cluster.x-k8s.io/adopt-deploymentzone: "dz1, dz2"
+	// if there are there deploymentzones available
```

Comment:
small typo: should be `if there are three...`

Reply:
Ack